test/runnable/test11742.d

```diff
@@ -0,0 +1,10 @@
+// https://issues.dlang.org/show_bug.cgi?id=11742
```

Comment:
Can this be merged into one of the "lots of test cases" files, like test42.d? The reason is then the autotester runs faster.